frames/src/main/java/com/checkout/frames/component/expirydate/ExpiryDateVisualTransformation.kt

```diff
@@ -15,46 +15,64 @@ internal class ExpiryDateVisualTransformation : VisualTransformation {
 
     private fun performExpiryDateFilter(text: AnnotatedString): TransformedText {
```

Comment:
are we able to add some extra test cases to check the transformed and original offsets so they will catch the issue if the logic has been changed in the future?

Reply:
Sure. Added test cases for it